source/dagger/gameplay/team_game/towers.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "mage.h"
+
+#include "core/core.h"
+#include "core/system.h"
+
+using namespace dagger;
+
+namespace ancient_defenders {
+
+    enum class SelectMode {
+        Spot = 1,
+        Tower = 2,
+    };
+
+    struct TowerPlacementInfo {
```

Comment:
@gavrilovmiroslav, take a look at this code. can u suggest a better way of doing this?

Reply:
Agreed, there shouldn't be a struct full of static things anywhere :D 

We'll talk about it when all the features are working, how does that sound, team?

Reply:
Sound good, this was the easiest way to get access to and modify all this data from a bunch of different spots.